src/manageiq_client/api.py

```diff
@@ -518,10 +516,17 @@ def reload(self):
             if action["name"] not in reloaded_actions:
                 reloaded_actions.append(action["name"])
                 setattr(self, action["name"], action_obj)
+
             setattr(self.__dict__[action["name"]],
                     action["method"].upper(),
                     action_obj)
 
+            # edit actions on entities can be performed using PATCH and PUT as well
+            if action["name"] == "edit" and isinstance(self._obj, Entity):
+                for edit_method in ("patch", "put"):
```

Comment:
this is fine but I'm just curious: why is this needed for `.edit` and only for it?
IIUC it happens automatically for e.g. `.delete.POST` & `.delete.DELETE`, because API gives us multiple "delete" actions?
Is it an API bug that multiple "edit" methods are not declared by API?

Reply:
Exactly as you said, "delete" with both POST and DELETE are listed in "actions" section of a resource. No actions with PUT and PATCH are listed there, but should be available for editing a resource wherever editing is supported.
It looks like a bug, I'm gonna make a BZ.